Tests/eppo/ConfigurationStoreTests.swift

```diff
@@ -30,6 +33,24 @@ final class ConfigurationStoreTests: XCTestCase {
         )
     }
     
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        // Clean up the configuration file
+        cleanUpConfigurationFile()
+        // Release the reference to ensure it's deallocated
+        configurationStore = nil
+    }
+    
+    private func cleanUpConfigurationFile() {
+        let fileManager = FileManager.default
+        let urls = fileManager.urls(for: .documentDirectory, in: .userDomainMask)
+        let fileURL = urls[0].appendingPathComponent("configuration.json")
+        
+        if fileManager.fileExists(atPath: fileURL.path) {
+            try? fileManager.removeItem(at: fileURL)
+        }
```

Comment:
nit: no need to check for existence if we're going to delete the file anyway
```suggestion
        try? fileManager.removeItem(at: fileURL)
```